test/dtypes/test_affine_quantized.py

```diff
@@ -180,8 +187,13 @@ def apply_uint6_weight_only_quant(linear):
     )
     @unittest.skipIf(not torch.cuda.is_available(), "Need CUDA available")
     def test_print_quantized_module(self, apply_quant):
+        print(apply_quant)
         linear = torch.nn.Linear(128, 256, dtype=torch.bfloat16, device="cuda")
-        ql = apply_quant(linear)
+        if isinstance(apply_quant, AOBaseConfig):
+            quantize_(linear, apply_quant)
+            ql = linear
+        else:
+            ql = apply_quant(linear)
```

Comment:
once we migrate all functions to configs we won't need this check anymore right? Should we add a TODO to remove it?